cms/djangoapps/contentstore/features/video_editor.py

```diff
@@ -146,6 +146,7 @@ def correct_video_settings(_step):
 
         # advanced
         [DISPLAY_NAME, 'Video', False],
+        ['Available Only on Web', 'False', False],
```

Comment:
to present this option along with the somewhat-similar "Video Download Allowed" field, I suggest the name "Video Available on Web Only"